pipenv/core.py

```diff
@@ -2184,44 +2184,66 @@ def inline_activate_virtualenv():
         )
 
 
+def _construct_run_command(project, command, args):
+    """Construct a command to run concatenating project and args.
+
+    ``command`` and/or the script from the ``Pipfile`` will be inserted before args.
+
+    Returns (executable, args for executable)"""
+    args = list(args or [])
+    # Script was found…
+    if command in project.scripts:
+        prefix_argv = project.scripts[command]
+    else:
+        # Separate out things that were passed in as a string.
+        prefix_argv = list(command.split())
```

Comment:
maybe `shlex.split` in case  we pass in complex command?

Reply:
Okay, I think no one would ever do `pipenv run "foo 'bar bla'".

Reply:
If they do, it should be parsed as-is. I mean, maybe they actually have a command called `"foo 'bar bla'"`, who knows.